src/index.js

```diff
@@ -24,13 +24,19 @@ require('./components/index');  // Register core components.
 var ANode = require('./core/a-node');
 var AEntity = require('./core/a-entity');  // Depends on ANode and core components.
 
-// Webvr polyfill configuration.
+// WebVR Polyfill configuration.
 window.hasNonPolyfillWebVRSupport = 'getVRDisplays' in navigator || 'getVRDevices' in navigator;
 window.WebVRConfig = {
   TOUCH_PANNER_DISABLED: true,
-  MOUSE_KEYBOARD_CONTROLS_DISABLED: true
+  MOUSE_KEYBOARD_CONTROLS_DISABLED: true,
+  DEFER_INITIALIZATION: true
 };
-require('../node_modules/webvr-libs/webvr-polyfill');
+var webvrPolyfill = require('../node_modules/webvr-libs/webvr-polyfill');
```

Comment:
this is precisely what I said in the PR:

https://github.com/borismus/webvr-polyfill/pull/33/files#r54689462
https://github.com/borismus/webvr-polyfill/pull/33/files#r54689463

unfortunately, it's since been merged